src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +128,20 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void helpTest() {
+        UI.events.registerEvent((NavigateToPageEventHandler) e -> url = e.url);
```

Comment:
Could you use an `AtomicReference<String>` for this instead? So it's kept local to the method.